comps/dataprep/milvus/langchain/config.py

```diff
@@ -3,16 +3,16 @@
 
 import os
 
-# Embedding model
-TEI_EMBEDDING_MODEL = os.getenv("EMBED_MODEL", "maidalun1020/bce-embedding-base_v1")
-# Embedding endpoints
+# Local Embedding model
+LOCAL_EMBEDDING_MODEL = os.getenv("LOCAL_EMBEDDING_MODEL", "maidalun1020/bce-embedding-base_v1")
```

Comment:
what if there is no "LOCAL_EMBEDDING_MODEL"?

Reply:
It will use the default value "maidalun1020/bce-embedding-base_v1".